### ISSUE TITLE
fastq or fastq.gz files for dada2 in R

### ISSUE THREAD

**Issue by maracashay** [maracashay/DAWG-Helpline#1]
Hi all, 

Just to start us off, I wanted to post something that I had gotten snagged up on when I first started using dada2 in R. 

You need to use .fastq files and your .fastq.gz files should NOT be in the same folder. DADA2 will give you an error when you try to read info from the .fastq files.

**Comment by emilyvansyoc** [comment]
Could you fix that within R by specifying the file type you're reading in with `list.files()`? That function takes a pattern as an argument where you could specify a regex for only fastq.

**Comment by maracashay** [comment]
Yes, you are absolutely right. Someone else pointed that out to me after making this post. I've since used the .fastq.gz files.
